howto/How_To_Use_OpenOCD_UbuntuEd

```diff
@@ -0,0 +1,67 @@
+---
+title: Openocd On Ubuntu 64
+redirect_from:
+- /Openocd.html
+---
+This How to covers, if I rip my usb how do I flash the Proffieboard or update it. there's a few things about this particular use case. In this article you will be using the ST-Linkv2 usb debugger that connects through single core and the operating system of your laptop or pc would be Ubuntu 64 bit or any linux distro that is 64 bit. 
+
+on a windows box these steps are not required because of the openocd files are located in the arduino ide folders. the main issue with Ubuntu 64 bit version is that the scripts on arduino require the 32 bit version and ubuntu has deprecated libudev.so.1, THERE IS A FIX THOUGH.
```

Comment:
This article is about linux, no need to talk about how it's done on windows, especially since I think the openocd for windows doesn't actually work...